src/Language/PureScript/Interactive.hs

```diff
@@ -90,22 +90,23 @@ make ms = do
 -- | Performs a PSCi command
 handleCommand
   :: (MonadReader PSCiConfig m, MonadState PSCiState m, MonadIO m)
-  => (String -> m ())
-  -> m ()
+  => (String -> m ()) -- ^ evaluate JS
```

Comment:
I wonder if we should put these into an MTL-style type class. The pile of wildcard binders below is getting a little silly :)